### PR TITLE
Standalone template for https://nsupdate.fedcloud.eu/

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
This will allow easier use of https://nsupdate.fedcloud.eu/ via IM dashboard

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Please see also: https://github.com/grycap/im-dashboard/pull/472

**Comment by micafer** [comment]
retest this please

File: templates/dydns_egi_update.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+              dydns_secret:
+                get_input: nsupdate_secret
+              new_ip:
+                get_attribute: [ simple_node, public_address, 0 ]
+      requirements:
+        - host: simple_node
+
+  outputs:
+    nsupdate_dns:
+      value: { get_input: [ minio_dns_console ] }
```
Comment (micafer) [inline templates/dydns_egi_update.yml:58]: This should be nsupdate_dns

Comment (sebastian-luna-valero) [reply templates/dydns_egi_update.yml]: true, thanks! amended in the following commit.

**Comment by micafer** [comment]
Added in the devel instance.

**Comment by sebastian-luna-valero** [comment]
Thanks!

I guess the below is not intended?

https://appsgrycap.i3m.upv.es/im-dashboard-dev/configure?selected_tosca=dydns_egi_update.properties.yml

With this icon:

image

**Pull request merged**